### PR TITLE
doc: add notes from session on project progression proposal

### PR DESCRIPTION
**Pull request by jorydotcom**
documents meeting from #69

### CODE REVIEW

File: meetings/breakout_meetings/2019-01-14-project-progression.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+# Project Progression Proposal - Issues for Stage 2 Acceptance
```
Comment (MylesBorins) [inline meetings/breakout_meetings/2019-01-14-project-progression.md]:
```suggestion
# Project Progression Proposal - Issues for Stage 2 Acceptance 2019-01-14
```

nit: we should include the date here

File: meetings/breakout_meetings/2019-01-14-project-progression.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Links
+
+* **Ref:** https://github.com/nodejs/bootstrap/blob/master/proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md
+* **Recording**: [coming soon]
```
Comment (MylesBorins) [inline meetings/breakout_meetings/2019-01-14-project-progression.md]:
```suggestion
* **Recording**:  https://www.youtube.com/watch?v=0hK1LGTZpP8
```

File: meetings/breakout_meetings/2019-01-14-project-progression.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+* Dave: Having true open governance requires enough people regularly participating, and enough diversity in the contribution base, etc. We need mentorship to exist.
+* Matteo: I can stand behind the JSF’s mentorship values 100%.
+
+### 15 mins - Defined Project Proposal Requirements, & explicit metrics for how to validate that those requirements have been met. (Chris H., Matteo, Jory)
+
+- do we require projects to keep public lists of orgs using their technology?
+
+
+### 10 mins - Clarification of Stage Definitions and Monikers (see also #44)
+
```
Comment (MylesBorins) [inline meetings/breakout_meetings/2019-01-14-project-progression.md]: Maybe add a note that we didn't get to this?

File: meetings/breakout_meetings/2019-01-14-project-progression.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+* Chris: if somebody wants to bring in their own CoC, maybe it makes sense to enforce values upon their project.
+* Dave: There aren’t a lot of places where it’s appropriate to centralize things, but this seems like one to me since it’s difficult to address this without that model.
+* Manil: If we’re expecting projects to have a CoC, they may not have the experience in building/doing this. We should give them training, and can only join if they do that. They should handle it well if they go through our training.
+* Dave: I agree - people who’re great at dealing with conflict resolution are best suited for this (more than folks who mostly write code). Training would help.
+* Michael: If nothing goes right, having an independent 3rd party to help with this seems right to me.
+
+### 5 mins - Clarification on what resources a Archived/Emeritus project will have/need
```
Comment (MylesBorins) [inline meetings/breakout_meetings/2019-01-14-project-progression.md]: Maybe add a note that we didn't get to this?

File: meetings/breakout_meetings/2019-01-14-project-progression.md (hunk 2 above)
Comment (jorydotcom) [reply meetings/breakout_meetings/2019-01-14-project-progression.md]: Thank you!

File: meetings/breakout_meetings/2019-01-14-project-progression.md (hunk 3 above)
Comment (jorydotcom) [reply meetings/breakout_meetings/2019-01-14-project-progression.md]: okiedok

File: meetings/breakout_meetings/2019-01-14-project-progression.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+* Dave: When you get to legal issues it gets a bit trickier, but if you look at the JSF lists for example - we do have a membership process. The period of time that you’re not a voting member could be very short if you want to be active and be a member of CPC. You can follow a best practice to get there.
+* Jory: My Concerns for these definitions is that you might be precluding these smaller projects from having a voice and being able to take on voting decisions.
+* Matteo: For me it seems a little bit complicated. If a project isn’t very active, would they be willing to invest time in building the foundation?
+* Jory: There’s a difference between thriving and small projects. There are upper limits to projects, though they are thriving, and that’s ok.
+* Michael: We’re trying to word it so anyone can participate in the regular work. There are multiple open paths to becoming voting members, if you’re big enough - get it, but if you’re smaller, you can get recognition as voting members by the many paths we have to get there.
```
Comment (mhdawson) [inline meetings/breakout_meetings/2019-01-14-project-progression.md]: I would change it to:

* Michael: We’re trying to word it so anyone can participate in the regular work. There are multiple open paths to becoming voting members, if the project is big enough there is direct representation. If the project is smaller there are still several paths to get there in the governance being proposed.

Comment (jorydotcom) [reply meetings/breakout_meetings/2019-01-14-project-progression.md]: done! thanks!

**Pull request merged**